src/sass/edu-benefits.scss

```diff
@@ -311,9 +311,17 @@ form.rjsf {
 .schemaform-back-buttons {
   margin-bottom: 7rem !important;
 }
+
 .schemaform-intro {
   padding-bottom: 1rem;
   margin-bottom: 7rem;
+
+  // To be moved into modules/_m-schemaform.scss when it's merged
+  .process {
```

Comment:
Gotcha. Can we add a separate class for this, instead of nesting the styles underneath `.schemaform-intro`? `schemaform-intro-process` or something.

I want us to avoid nesting as much as possible. It leads to things like the issue with progress-box above.